scripts/SANS/sans/gui_logic/presenter/property_manager_service.py

```diff
@@ -0,0 +1,107 @@
+from mantid.api import Algorithm
+from mantid.kernel import (PropertyManagerProperty, PropertyManagerDataService)
+from sans.state.state_base import create_deserialized_sans_state_from_property_manager
+
+
+class PropertyManagerService(object):
+    sans_property_manager_prefix = "SANS_PROPERTY_MANAGER_THIS_NEEDS_TO_BE_UNIQUE_"
+
+    class ConverterAlgorithm(Algorithm):
```

Comment:
@AntonPiccardoSelg You are correct, we can only assign via a string to the property directly so this is the cleanest way to do that at the moment.